server/index.js

```diff
@@ -28,16 +28,12 @@ app.get('/healthcheck', (req, res) => {
   res.send('OK')
 })
 
-app.use(csp({directives: customCsp}))
 app.use(userAuth)
```

Comment:
If we wanted to serve the static files without auth, could we just attach a separate express.static() middleware before the userAuth? I'd prefer to put it behind some sort of switch for security considerations (see overall comments)